ui/app/peers/create/configuration/helpers/pg.ts

```diff
@@ -0,0 +1,36 @@
+import { PeerSetter } from '../types';
+
+export const postgresSetting = [
+  {
+    label: 'Host',
+    stateHandler: (value: string, setter: PeerSetter) =>
+      setter((curr) => ({ ...curr, host: value })),
+  },
+  {
+    label: 'Port',
+    stateHandler: (value: string, setter: PeerSetter) =>
+      setter((curr) => ({ ...curr, port: parseInt(value, 10) })),
+    type: 'number', // type for textfield
```

Comment:
The Port should have a min max range, it should not allow negative values. 

![image](https://github.com/PeerDB-io/peerdb/assets/39798643/4787f017-81c8-4184-9da1-34a12ab7b85e)


The way PgConfig and Config form components are setup there is no way to send this info from the settings props.

Reply:
We could prevent typing of non-numbers but right now I personally think it's fine. Happy to discuss though